packages/driver/src/dom/transform.ts

```diff
@@ -44,15 +48,17 @@ const extractTransformInfoFromElements = ($el: any, list: TransformInfo[] = []):
   return extractTransformInfoFromElements($parent, list)
 }
 
-const extractTransformInfo = ($el): TransformInfo => {
+const extractTransformInfo = ($el): TransformInfo | null => {
   const el = $el[0]
   const style = getComputedStyle(el)
 
-  return {
-    backfaceVisibility: style.getPropertyValue('backface-visibility') as BackfaceVisibility,
-    transformStyle: style.getPropertyValue('transform-style') as TransformStyle,
-    transform: style.getPropertyValue('transform'),
-  }
+  return style.getPropertyValue('backface-visibility') !== ''
+    ? {
+      backfaceVisibility: style.getPropertyValue('backface-visibility') as BackfaceVisibility,
+      transformStyle: style.getPropertyValue('transform-style') as TransformStyle,
+      transform: style.getPropertyValue('transform'),
+    }
+    : null
```

Comment:
```suggestion
  const backfaceVisibility = style.getPropertyValue('backface-visibility')
  
  // this handles an edge case from frameworks like `vue-fragment` 
  // that modify the `parentNode` of the element
  // @see https://github.com/cypress-io/cypress/pull/6787
  // @see https://github.com/cypress-io/cypress/issues/6745
  if (backfaceVisibility === '') {
    return null
  }
  
  return {
    backfaceVisibility as BackfaceVisibility,
    transformStyle: style.getPropertyValue('transform-style') as TransformStyle,
    transform: style.getPropertyValue('transform'),
  }
```

Reply:
just want this change to clarify on why its necessary, and link out to the additional bits

Reply:
Done.